### DISCUSSION TITLE
How to configure a Private Key

### DISCUSSION THREAD

**Discussion by zhaokaixin** [apache/camel-karavan#878] [Q&A]
I am using the as2 component and I need to configure the Private Key, can someone tell me how to set it up?
![image](https://github.com/apache/camel-karavan/assets/22127268/a23f7db0-f39c-4a5a-a2f3-685c067ccd64)

**Comment by davsclaus** [comment]
https://issues.apache.org/jira/browse/CAMEL-21863
